editor/src/templates/property-controls-info.tsx

```diff
@@ -53,14 +53,18 @@ const initPropertyControlsWorker = () => {
         await createBundle(bundlerWorker, emptyTypeDefinitions, projectContents)
       ).buildResult
 
+      // Mutating the evaluation cache.
+      for (const fileToDelete of model.updatedAndReverseDepFilenames) {
+        delete evaluationCache[fileToDelete]
```

Comment:
this mirrors the same behavior as in code-file.ts, if the user updates the propertyControls, we need to delete the old file from the evaluationCache, this does that

Reply:
Is it worth adding that as a comment into the code?